onboarding_auth/onboardingAuthDomain/src/main/java/com/alexafit/onboardingauthdomain/repository/OnboardingAuthRepository.kt

```diff
@@ -2,13 +2,16 @@ package com.alexafit.onboardingauthdomain.repository
 
 import com.alexafit.onboardingauthdomain.model.remote.LoginUser
 import com.alexafit.onboardingauthdomain.model.remote.RegisterUser
+import kotlinx.coroutines.flow.Flow
 
 interface OnboardingAuthRepository {
```

Comment:
I would rename it to Auth instead of OnboardingAuth

Usually, Auth is a separate feature than Onboarding (+ Tasky doesnt have Onboarding)